src/targets/gpu/mlir.cpp

```diff
@@ -1258,7 +1258,7 @@ void dump_mlir_to_mxr(module m,
     }
     auto name = compute_dump_name(m, ".mxr");
     std::cout << "Dumping MXR file to: " << name << std::endl;
-    auto f    = location / name;
+    auto f = location / name;
```

Comment:
Will this work on Windows? a forward slash `/`?

Reply:
Yes it will.